_pkgdown.yml

```diff
@@ -1 +1,2 @@
-destination: docs
+template:
+  bootstrap: 5
```

Comment:
Thanks.

What does this do?

Reply:
this adds a search bar and makes the website look a bit less dull in my opinion. You can always revert later if you prefer the old way! see https://dplyr.tidyverse.org for an example